crates/rpc-types-eth/src/filter.rs

```diff
@@ -800,135 +1013,6 @@ where
     }
 }
 
-/// Support for matching [Filter]s
-#[derive(Debug, Default)]
-pub struct FilteredParams {
-    /// The original filter, if any
-    pub filter: Option<Filter>,
-}
-
-impl FilteredParams {
-    /// Creates a new wrapper type for a [Filter], if any with flattened topics, that can be used
-    /// for matching
-    pub fn new(filter: Option<Filter>) -> Self {
-        filter.map_or_else(Default::default, |filter| Self { filter: Some(filter) })
-    }
-
-    /// Returns the [BloomFilter] for the given address
-    pub fn address_filter(address: &FilterSet<Address>) -> BloomFilter {
-        address.to_bloom_filter()
-    }
-
-    /// Returns the [BloomFilter] for the given topics
-    pub fn topics_filter(topics: &[FilterSet<B256>]) -> Vec<BloomFilter> {
-        topics.iter().map(|t| t.to_bloom_filter()).collect()
-    }
-
-    /// Returns `true` if the bloom matches the topics
-    pub fn matches_topics(bloom: Bloom, topic_filters: &[BloomFilter]) -> bool {
-        if topic_filters.is_empty() {
-            return true;
-        }
-
-        // for each filter, iterate through the list of filter blooms. for each set of filter
-        // (each BloomFilter), the given `bloom` must match at least one of them, unless the list is
-        // empty (no filters).
-        for filter in topic_filters {
-            if !filter.matches(bloom) {
-                return false;
-            }
-        }
-        true
-    }
-
-    /// Returns `true` if the bloom contains one of the address blooms, or the address blooms
-    /// list is empty (thus, no filters)
-    pub fn matches_address(bloom: Bloom, address_filter: &BloomFilter) -> bool {
-        address_filter.matches(bloom)
-    }
-
-    /// Returns true if the filter matches the given block number
-    pub fn filter_block_range(&self, block_number: u64) -> bool {
```

Comment:
what's the equivalent of this now?

because we need this on the server side

Reply:
- `matches_block_range` for numbers 
- `matches_block_hash` for hashes
- `matches_block` for numhashes